assets/caniuse.css

```diff
@@ -16,6 +16,30 @@ button.caniuse-cell {
   position: relative;
 }
 
+/* handle case when printing */
+@media print {
+  .caniuse-cell.y::before {
+    background: #8bc34a;
+    content: "&#10004;";
```

Comment:
It's preferable just to put the actual emoji here, otherwise it's difficult to know what's actually going to shown.

Reply:
The above two comments apply below too.